Redfish.md

```diff
@@ -770,8 +770,9 @@ other.
 - SerialNumber
 - SparePartNumber
 - Status
+- Slots
```

Comment:
isn't this OEM? You can either leave off.. other OEM aren't there or put under OEM/IBM/Slots

Reply:
Got it. 
Removed the change of Redfish.md.